### PR TITLE
samples: refactor automl beta test for importDataset

### LINKED ISSUE #3573: com.example.automl.ImportDatasetTest: testImportDataset failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: fead83412a59e810bf546ddc95a7e96d359461da
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/a83732cf-b048-4853-aeae-cb6bc9d69fe7), [Sponge](http://sponge2/a83732cf-b048-4853-aeae-cb6bc9d69fe7)
status: failed
<details><summary>Test output</summary><br><pre>java.util.concurrent.ExecutionException: 
com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN7572588368656072704.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN2912481507408871424
Call ListOperations API to learn more details.
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:564)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:545)
	at com.google.common.util.concurrent.FluentFuture$TrustedFuture.get(FluentFuture.java:86)
	at com.google.common.util.concurrent.ForwardingFuture.get(ForwardingFuture.java:62)
	at com.google.api.gax.longrunning.OperationFutureImpl.get(OperationFutureImpl.java:127)
	at com.example.automl.DeleteDataset.deleteDataset(DeleteDataset.java:44)
	at com.example.automl.ImportDatasetTest.tearDown(ImportDatasetTest.java:91)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
Caused by: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN7572588368656072704.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN2912481507408871424
Call ListOperations API to learn more details.
	at com.google.api.gax.rpc.ApiExceptionFactory.createException(ApiExceptionFactory.java:59)
	at com.google.api.gax.grpc.GrpcApiExceptionFactory.create(GrpcApiExceptionFactory.java:72)
	at com.google.api.gax.grpc.GrpcApiExceptionFactory.create(GrpcApiExceptionFactory.java:60)
	at com.google.api.gax.grpc.GrpcExceptionCallable$ExceptionTransformingFuture.onFailure(GrpcExceptionCallable.java:97)
	at com.google.api.core.ApiFutures$1.onFailure(ApiFutures.java:68)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1050)
	at com.google.common.util.concurrent.DirectExecutor.execute(DirectExecutor.java:30)
	at com.google.common.util.concurrent.AbstractFuture.executeListener(AbstractFuture.java:1176)
	at com.google.common.util.concurrent.AbstractFuture.complete(AbstractFuture.java:969)
	at com.google.common.util.concurrent.AbstractFuture.setException(AbstractFuture.java:760)
	at io.grpc.stub.ClientCalls$GrpcFuture.setException(ClientCalls.java:563)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:533)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:434)
	at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:763)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:742)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN7572588368656072704.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN2912481507408871424
Call ListOperations API to learn more details.
	at io.grpc.Status.asRuntimeException(Status.java:533)
	... 13 more
</pre></details>

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes #3573 

- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java [hunk 1]
```diff
@@ -60,22 +65,31 @@ public static void checkRequirements() {
   }
 
   @Before
-  public void setUp() throws IOException {
+  public void setUp()
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
```
Comment (lesv) [inline automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java:69]: Nice

File: automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java [hunk 2]
```diff
@@ -88,15 +102,30 @@ public void setUp() throws IOException {
   @After
   public void tearDown() throws InterruptedException, ExecutionException, IOException {
     // Delete the created dataset
-    DeleteDataset.deleteDataset(PROJECT_ID, datasetId);
+
+    // DeleteDataset.deleteDataset(PROJECT_ID, datasetId);
     System.setOut(null);
```
Comment (lesv) [inline automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java:107]: Not your change - we should actually set this to the saved stdOut - ideally saved around L85 (now L97) `System.out`

Comment (munkhuushmgl) [reply automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java:107]: Oh forgot to uncomment deleteDataset part, it should be uncommented i was just testing something out.

Comment (munkhuushmgl) [reply automl/beta/src/test/java/com/example/automl/ImportDatasetTest.java]: I will make a note of System.out part.

**Comment by lesv** [comment]
@munkhuushmgl failing java 11

**Pull request merged**